lib/BC/SleighLifter.cpp

```diff
@@ -484,10 +495,97 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
     return LiftStatus::kLiftedInstruction;
   }
 
+  LiftStatus LiftFloatUnop(llvm::IRBuilder<> &bldr, OpCode opc,
+                           VarnodeData *outvar, VarnodeData input_var) {
```

Comment:
I would consider taking `outvar` by `&` in case it cannot be `nullptr` (if it can be done, I see if passed along to some other calls).

Reply:
I think I'm going to keep it as is since it gets passed down to LiftStoreIntoOutParam which is a generic function for requiring an outparam to store into and handles the nullcheck. Alternatively, I could take a reference and make it back into a pointer but that seems eh